blocks/api/serializer.js

```diff
@@ -37,8 +39,18 @@ export function getSaveContent( save, attributes ) {
 		}
 	}
 
+	// Adding a generic classname
+	const addClassnameToElement = ( element ) => {
+		if ( ! element || ! isObject( element ) ) {
+			return element;
+		}
+		const className = classnames( element.props.className, `wp-block-${ kebabCase( blockName ) }` );
```

Comment:
How so? Core blocks would have no namespace and plugin blocks will have the plugin namespace.